src/main/java/com/palantir/gradle/graal/GraalExtension.java

```diff
@@ -90,4 +94,19 @@ public final void graalVersion(String value) {
         return graalVersion;
     }
 
+
+    public final ListProperty<String> getOptions() {
+        return this.options;
+    }
+
+    /**
+     * Add option from https://github.com/oracle/graal/blob/master/substratevm/OPTIONS.md.
+     */
+    public final void option(String option) {
+        if (option.trim().startsWith("-H:Name=")) {
+            throw new GradleException("Use 'outputName' instead of '" + option + "'");
+        }
+        this.options.add(option);
```

Comment:
as a future-looking thing we might want to add a check here to ensure the option starts with a '-'.